app-server/app/src/test/scala/organization/OrganizationSpec.scala

```diff
@@ -14,15 +23,133 @@ class OrganizationSpec extends WordSpec
     with DBSpec {
   implicit val ec = system.dispatcher
   implicit def database = db
+  implicit def default(implicit system: ActorSystem) = RouteTestTimeout(DurationInt(20).second)
+  val now = new java.util.Date()
```

Comment:
Probably makes sense to create a trait that handles creating a json web token since this set of code seems to be nearly duplicated in multiple spots